README.md

```diff
@@ -185,6 +185,8 @@ All settings are prefixed with `cSpell:` or `spell-checker:`.
 
 It is possible to disable / enable the spell checker by adding comments to your code.
 
+> Note: the comment characters for your file type should be used. The important part is `cSpell:disable` and `cSpell:enable`, or the equivalents from the following sections.
```

Comment:
I'm not sure if this line is needed if the comment characters are removed from the list below.